modules/gis/src/types.ts

```diff
@@ -0,0 +1,79 @@
+import {GeoJsonProperties} from 'geojson';
+
+type TypedArray =
+  | Int8Array
+  | Uint8Array
+  | Int16Array
+  | Uint16Array
+  | Int32Array
+  | Uint32Array
+  | Uint8ClampedArray
+  | Float32Array
+  | Float64Array;
+
+export type BinaryAttribute = {value: TypedArray; size: number};
+export type BinaryGeometryType = 'Point' | 'LineString' | 'Polygon';
+
+type NumericProps = {[key: string]: BinaryAttribute};
+
+/**
+ * GIS category loader format
+ */
+export type DataFormatGIS = {
+  category: 'gis';
+  schema: object; /** `Schema` - Apache Arrow style schema  */
+  data: object /** Data is formatted according to `options.gis.format` */
```

Comment:
Seems like it would be helpful (maybe a future PR) to have `DataFormatBinaryGIS` and `DataFormatGeoJsonGIS` so that this `data` object could have a better type.